cpu/stm32f3/vectors.c

```diff
@@ -18,362 +18,227 @@
-    isr_adc1_2,             /* [18] ADC1 & ADC2 Interrupts */
-    isr_usb_hp_can_tx,      /* [19] USB Device High Priority or CAN TX Interrupts */
-    isr_usb_lp_can_rx0,     /* [20] USB Device Low Priority or CAN RX0 Interrupts */
-    isr_can_rx1,            /* [21] CAN RX1 Interrupt */
-    isr_can_sce,            /* [22] CAN SCE Interrupt */
-    isr_exti,               /* [23] External Line[9:5] Interrupts */
-    isr_tim1_brk_tim15,     /* [24] TIM1 Break and TIM15 Interrupts */
-    isr_tim1_up_tim16,      /* [25] TIM1 Update and TIM16 Interrupts */
-    isr_tim1_trg_com_tim17, /* [26] TIM1 Trigger and Commutation and TIM17 Interrupt */
-    isr_tim1_cc,            /* [27] TIM1 Capture Compare Interrupt */
-    isr_tim2,               /* [28] TIM2 global Interrupt */
+    /* shared vectors for all family members */
+    [ 0] = isr_wwdg,                 /* [ 0] Window WatchDog Interrupt */
+    [ 1] = isr_pvd,                  /* [ 1] PVD through EXTI Line detection Interrupt */
+    [ 2] = isr_tamp_stamp,           /* [ 2] Tamper and TimeStamp interrupts through the EXTI line 19 */
+    [ 3] = isr_rtc_wkup,             /* [ 3] RTC Wakeup interrupt through the EXTI line 20 */
+    [ 4] = isr_flash,                /* [ 4] FLASH global Interrupt */
+    [ 5] = isr_rcc,                  /* [ 5] RCC global Interrupt */
+    [ 6] = isr_exti,                 /* [ 6] EXTI Line0 Interrupt */
+    [ 7] = isr_exti,                 /* [ 7] EXTI Line1 Interrupt */
+    [ 8] = isr_exti,                 /* [ 8] EXTI Line2 Interrupt and Touch Sense Controller Interrupt */
+    [ 9] = isr_exti,                 /* [ 9] EXTI Line3 Interrupt */
+    [10] = isr_exti,                 /* [10] EXTI Line4 Interrupt */
+    [11] = isr_dma1_channel1,        /* [11] DMA1 Channel 1 Interrupt */
+    [12] = isr_dma1_channel2,        /* [12] DMA1 Channel 2 Interrupt */
+    [13] = isr_dma1_channel3,        /* [13] DMA1 Channel 3 Interrupt */
+    [14] = isr_dma1_channel4,        /* [14] DMA1 Channel 4 Interrupt */
+    [15] = isr_dma1_channel5,        /* [15] DMA1 Channel 5 Interrupt */
+    [16] = isr_dma1_channel6,        /* [16] DMA1 Channel 6 Interrupt */
+    [17] = isr_dma1_channel7,        /* [17] DMA1 Channel 7 Interrupt */
+    [21] = isr_can_rx1,              /* [21] CAN RX1 Interrupt */
+    [22] = isr_can_sce,              /* [22] CAN SCE Interrupt */
+    [23] = isr_exti,                 /* [23] External Line[9:5] Interrupts */
+    [24] = isr_tim1_brk_tim15,       /* [24] TIM1 Break and TIM15 Interrupts */
+    [25] = isr_tim1_up_tim16,        /* [25] TIM1 Update and TIM16 Interrupts */
+    [26] = isr_tim1_trg_com_tim17,   /* [26] TIM1 Trigger and Commutation and TIM17 Interrupt */
+    [27] = isr_tim1_cc,              /* [27] TIM1 Capture Compare Interrupt */
+    [28] = isr_tim2,                 /* [28] TIM2 global Interrupt */
+    [32] = isr_i2c1_er,              /* [32] I2C1 Error Interrupt */
```

Comment:
This is very strange, some isr are not there any more: 31, 33, 34. There might be a bug in the script that generates the table ?

Reply:
don't panic @aabadie 😲  the above are generic ISRs for all F3 MCUs, you'll find your missing 31, 33, 34 and more below in the MCU specific parts.

Reply:
mhm, maybe you're still a bit right, I see 34 below but 31, and 33 are now missing? @haukepetersen according to e.g., [stm32f334x8](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32f3/include/vendor/stm32f334x8.h#L126) there should be `[31] = isr_i2c1_ev`

Reply:
indeed, I found 34 after commenting but not the others. So there's an issue I think